src/style.css

```diff
@@ -0,0 +1,88 @@
+* {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 32px 16px 16px;
+}
+
+.img {
+  /* padding: 32px 19px 0 19px; */
+}
+
+.text {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  text-align: inherit;
+  padding: 40px 0 0;
+  color: #060b35;
+}
+
+.card__code {
+  padding: 4px 0 0;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin: 2px;
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  padding: 16px 0 0;
+}
+
+.stars :nth-child(-n +4) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars__review {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 18px;
+  color: #060b35;
+  padding-left: 17px;
+}
```

Comment:
Check please correct styles for price text and value in figma and compare with your results
<img width="424" alt="image" src="https://user-images.githubusercontent.com/30314756/182901935-b27fca65-f08a-4b44-8c76-d7798845271b.png">

Reply:
this styles are from another element , review:5 which is located in one row with the stars